ajax/libs/pickmeup/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "pickmeup",
+  "filename": "pickmeup.min.js",
+  "description": "Really simple, powerful, customizable and lightweight standalone datepicker",
+  "keywords": [
+    "datepicker",
+    "date",
+    "calendar"
+  ],
+  "version": "3.0.11",
+  "homepage": "https://cleverstyle.org/",
+  "author": {
+    "name": "Nazar Mokrynskyi",
+    "email": "nazar@mokrynskyi.com",
+    "url": "https://cleverstyle.org"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/nazar-pc/PickMeUp.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/nazar-pc/PickMeUp.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "{js,css}/?(jquery.)pickmeup*.+(js|css|map)"
+        ]
+      },
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      }
+    ]
+  },
+  "license": "MIT"
```

Comment:
Updated to `0BSD` 29 days ago.
See https://github.com/nazar-pc/PickMeUp/blob/master/package.json#L17